scripts/zkai_d128_range_policy_discipline_gate.py

```
@@ -0,0 +1,736 @@
+                    "surface",
+                    "baseline_accepted",
+                    "mutated_accepted",
+                    "rejected",
+                    "rejection_layer",
+                    "error",
+                },
+                f"mutation case {index}",
+            )
+            expect_equal(case.get("mutation"), expected_case["mutation"], f"mutation case {index} name")
+            expect_equal(case.get("surface"), expected_case["surface"], f"mutation case {index} surface")
+            expect_equal(case.get("baseline_accepted"), True, f"mutation case {index} baseline")
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)